test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -532,14 +536,14 @@ UCS_TEST_P(test_ucp_sockaddr_with_rma_atomic, wireup) {
      * features are RMA/ATOMIC. With these features, need to make sure that
      * there is a lane for ucp-wireup (an am_lane should be created and used) */
     UCS_TEST_MESSAGE << "Testing " <<
-        ucs::sockaddr_to_str((const struct sockaddr*)&test_addr);
+        ucs::sockaddr_to_str(&test_addr.get_sock_addr());
 
-    start_listener(cb_type(), (const struct sockaddr*)&test_addr);
+    start_listener(cb_type(), &test_addr.get_sock_addr(), test_addr.get_addr_size());
```

Comment:
use get_sock_addr_ptr (in multiple places)

Reply:
get_sock_addr_ptr is private

Reply:
can make it public, it seems useful